timbermill-client/src/main/java/com/datorama/oss/timbermill/TimberLogger.java

```diff
@@ -50,13 +50,17 @@ public static String getCurrentTaskId() {
 	}
 
 	public static String start(String name) {
-		return start(name, null);
+		return start(name, (LogParams) null);
```

Comment:
Added this casting because now there are two methods with two parameters and the compiler couldn't decide which should be triggered.

Reply:
@roiorlan Let's make it call start(String name, String parentTaskId, LogParams logParams)